packages/obojobo-document-engine/ObojoboDraft/Sections/Assessment/post-assessment/editor.js

```diff
@@ -0,0 +1,188 @@
+/* eslint no-alert: 0 */
+import React from 'react'
+import { Block } from 'slate'
+import { CHILD_REQUIRED, CHILD_TYPE_INVALID } from 'slate-schema-violations'
+
+// A single score action
+const SCORE_NODE = 'ObojoboDraft.Sections.Assessment.ScoreAction'
+// The whole array of score actions
+const ACTIONS_NODE = 'ObojoboDraft.Sections.Assessment.ScoreActions'
+const PAGE_NODE = 'ObojoboDraft.Pages.Page'
+
+import Page from '../../../Pages/Page/editor'
+
+class Score extends React.Component {
+	constructor(props) {
+		super(props)
+	}
+
+	changeRange() {
+		const dataFor = this.props.node.data.get('for')
+		const newRange = window.prompt('Enter the new range:', dataFor) || dataFor
+
+		// TODO Validate range
```

Comment:
TODO

Reply:
Removed the TODO statement, validation is added in https://github.com/ucfopen/Obojobo/pull/671